app/controllers/organisers_controller.rb

```diff
@@ -4,6 +4,21 @@ class OrganisersController < ApplicationController
   def index
     @organisers = User.where(role: 1)
   end
+  
+
+  def create
+    @user = User.find_by(email: params[:email])
+    if @user.blank? || (@user.role != "organizer")
```

Comment:
you're not saving an user here. why? it's supposed to be 'create' action ;) Also why you're checking the role? It doesn't matter whether user with "abc@abc.com" exists but his role is "mentee". Creating organizer in such case should be impossible as based on the role user will have access to different page. Unless we want people to be able to be both organizer and mentee but than how will we decide where to redirect them and what kind of links to show them? I'm for using one email address per role.